### PR TITLE
Splash Screen Caching

### PR DESCRIPTION
**Pull request by sebastian24**
https://jira.spilgames.com/browse/NAT-2965

### CODE REVIEW

File: spilsdk/src/main/java/com/spilgames/spilsdk/splashscreen/SplashscreenManager.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+     */
+    public void showSplashscreen() {
+        if (delayedSplashscreen == null) {
+            SpilSdk.getInstance(context).getSplashScreenCallbacks().splashScreenNotAvailable();
+
+            return;
+        }
+
+        try {
+            Intent intent = new Intent(context, WebViewActivity.class);
```
Comment (thijsterlouw) [inline spilsdk/src/main/java/com/spilgames/spilsdk/splashscreen/SplashscreenManager.java]: this seems to resemble the code around line 109 a lot (both to show a splashscreen). Could be refactored a bit with additional benefit of having the same exception handling

File: spilsdk/src/main/java/com/spilgames/spilsdk/SpilSdk.java [hunk 2]
```diff
@@ -1998,6 +2007,13 @@ public void resetData() {
         //Resetting Asset Bundles
         AssetBundlesManager.getInstance(context).resetAssetBundlesData();
         requestAssetBundles();
+
+        //Resetting Daily Bonus
+        DailyBonusManager.getInstance(context).resetDailyBonusConfig();
```
Comment (thijsterlouw) [inline spilsdk/src/main/java/com/spilgames/spilsdk/SpilSdk.java]: is this change related to the dailybonus problems?

Comment (sebastian24) [reply spilsdk/src/main/java/com/spilgames/spilsdk/SpilSdk.java]: No, this is just to make extra sure that no daily bonus values are kept before a resetting happens. By default daily bonus is requested by the developer.

File: spilsdk/src/main/java/com/spilgames/spilsdk/splashscreen/SplashscreenManager.java (hunk 1 above)
Comment (sebastian24) [reply spilsdk/src/main/java/com/spilgames/spilsdk/splashscreen/SplashscreenManager.java]: will add a general method for it
